README.md

```diff
@@ -28,34 +28,47 @@ In this project you will create a login page and request a token from the server
 
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
-- [ ] Explain what a token is used for.
-- [ ] What steps can you take in your web apps to keep your data secure?
-- [ ] Describe how web servers work.
-- [ ] Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
+- [x] Explain what a token is used for.
 
+      A unique identifier for an application requesting access
+
+- [x] What steps can you take in your web apps to keep your data secure?
+
+      make sure that authentication is in place
+
+- [x] Describe how web servers work.
+
+      a server returns web page information from a web provider
+
+- [x] Which HTTP methods can be mapped to the CRUD acronym that we use when interfacing with APIs/Servers.
+
+      POST  Creates a new resource.
+      GET  Retrieves a resource.
+      PUT  Updates an existing resource.
+      DELETE  Deletes a resource.
```

Comment:
These are all correct! Which letter from the acronym `CRUD` would these match up with? (Example, D = delete, which matches with the `DELETE` request)